inc/Post_Thumbnails/Component.php

```diff
@@ -40,4 +42,12 @@ public function initialize() {
 	public function action_add_post_thumbnail_support() {
 		add_theme_support( 'post-thumbnails' );
 	}
+
+	/**
+	 * Adds custom image sizes.
+	 */
+	public function action_add_image_sizes() {
+		set_post_thumbnail_size( 720, 308, true );
```

Comment:
Use `add_image_size()` instead. Many plugins expect there to be a standard 150 x 150 thumbnail available. `set_post_thumbnail_size()` changes this default. Best-practice is to always preserve the standard core sizes and include custom sizes where other sizes are required.

Reply:
Thank you for the review. I have updated the code and squashed the commits.